packages/react-components/react-nav-preview/src/components/NavCategory/useNavCategory.ts

```diff
@@ -12,20 +12,15 @@ import type { NavCategoryProps, NavCategoryState } from './NavCategory.types';
  * @param ref - reference to root HTMLDivElement of NavCategory
  */
 export const useNavCategory_unstable = (props: NavCategoryProps, ref: React.Ref<HTMLDivElement>): NavCategoryState => {
+  const { value, children } = props;
+
+  const { openItems } = useNavContext_unstable();
+
+  const open: boolean = openItems?.includes(value);
```

Comment:
this is more of a super nit, but I would try to keep the general destructs, single variable that come from external sources and context values in a block of code. 

 ```suggestion
  const { value, children } = props;
  const { openItems } = useNavContext_unstable();
  const open: boolean = openItems?.includes(value);
```